### DISCUSSION TITLE
Stats and Logs are empty

### DISCUSSION THREAD

**Discussion by djpinger** [tphakala/birdnet-go#122] [Q&A]
Good afternoon!  I LOVE the project.  Been running Birdnet-Pi, but I was tired of using an rpi for this.  One thing I am encountering is that the logs and stats sections always say "Nothing here yet..".  Is that expected?  Here is my docker-compose.yml file:

`name: birdnet-go
services:
    birdnet-go:
        tty: true
        stdin_open: true
        ports:
            - 1234:8080
        environment:
            - ALSA_CARD=1
            - TZ=America/New_York
        devices:
            - /dev/snd
        volumes:
            - ./config:/config
            - ./data:/data
        image: ghcr.io/tphakala/birdnet-go:latest`

Thanks for everything!

**Comment by tphakala** [comment]
Hello, thanks for reaching out. Web UI still has unfinished parts, like stats and logs, I am very bad at frontend coding so progress on at that part has been slow. I hope to get more experienced frontend developers to join and contribute but I just need to find time to write some documentation first.

**Comment by djpinger** [comment]
Thanks!  I'm looking at things now to see what is needed and seeing if any folks I know want to help :).
